setup-devel.yaml

```diff
@@ -20,6 +20,8 @@ services:
                 PIP_INSTALL_ODOO: "false"
                 CLEAN: "false"
                 COMPILE: "false"
+        networks:
+            - public
```

Comment:
Public for this?

Reply:
Yes, in this case the running container needs to connect to the network to download git code from github usually.